src/server/common.h

```diff
@@ -59,14 +60,14 @@ struct LockTagOptions {
   char close_locktag = '}';
   unsigned skip_n_end_delimiters = 0;
   std::string prefix;
-};
 
-struct KeyLockArgs {
-  static LockTagOptions GetLockTagOptions();
+  // Returns the tag according to the rules defined by this options object.
+  std::string_view Tag(std::string_view key) const;
 
-  // Before acquiring and releasing keys, one must "normalize" them via GetLockKey().
-  static std::string_view GetLockKey(std::string_view key);
+  static const LockTagOptions& instance();
```

Comment:
Maybe `default()` is a better name here?

Reply:
default implies construction, since it's a thread-local singleton, I think instance or tlocal are better names. maybe tlocal is better just because of consistency